api/src/lib.rs

```diff
@@ -31,11 +34,25 @@ pub async fn app(pool: PgPool) -> Router {
             "/api/projects/:id",
             get(handlers::get_project).put(handlers::update_project),
         )
-        .route("/api/token_test", get(handlers::token_test))
         .layer(
             ServiceBuilder::new()
                 .layer(TraceLayer::new_for_http())
-                .layer(CorsLayer::permissive())
+                .layer(
+                    CorsLayer::new()
+                        .allow_credentials(true)
+                        .allow_methods(vec![Method::GET, Method::POST, Method::PUT, Method::DELETE])
+                        .allow_origin(Origin::list(vec![
+                            "http://localhost:8000".parse().expect("parse origin"),
+                            "https://api.dev.swissgeol.ch"
+                                .parse()
+                                .expect("parse origin"),
+                            "https://api.int.swissgeol.ch"
+                                .parse()
+                                .expect("parse origin"),
+                            "https://api.swissgeol.ch".parse().expect("parse origin"),
```

Comment:
You should also allow the likes of https://dev.swissgeol.ch

Reply:
Including `viewer.swissgeol.ch`, `int.swissgeol.ch`, `review.swissgeol.ch`?

Reply:
Addressed in https://github.com/swissgeol/ngm/pull/855/commits/881d27ec98c5548d115a3cc5f42e2dbc4142122f